AGENTS.md

```
@@ -0,0 +1,84 @@
+   ---
+   ```
+3. Add optional bundled resources:
+   - `scripts/` - Executable code
+   - `references/` - Documentation references
+   - `assets/` - Templates and files for output
+
+See the [skill-creator](.agent/skills/skill-creator/SKILL.md) skill for detailed guidance on creating effective skills.
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)